cmd/cli/main.go

```diff
@@ -42,7 +41,7 @@ func init() {
 	flag.StringVar(&svcID, "service", svcDefault, svcIDDescription)
 	flag.StringVar(&svcID, "s", svcDefault, svcIDDescription+" (short)")
 
-	flag.StringVar(&uid, "uid", "", uidDescription)
+	flag.StringVar(&name, "name", "", nameDescription)
```

Comment:
just a reminder that this (and the change from `-v` to `--version` below) will need changes in docs

Reply:
Yep. Thanks